sdk/opa_test.go

```diff
@@ -5,12 +5,12 @@
 package sdk_test
 
 import (
-	"bytes"
```

Comment:
⚠️ This won't merge cleanly after https://github.com/open-policy-agent/opa/pull/4121 was merged. Fun thing is that github doesn't seem to notice.

I'll fix it when rebasing.